EventFilter/SiPixelRawToDigi/plugins/SiPixelRawToDigi.cc

```diff
@@ -97,6 +98,9 @@ SiPixelRawToDigi::SiPixelRawToDigi( const edm::ParameterSet& conf )
     usePhase1 = config_.getParameter<bool> ("UsePhase1");
     if(usePhase1) edm::LogInfo("SiPixelRawToDigi")  << " Use pilot blade data (FED 40)";
   }
+  //CablingMap could have a label //Tav
+  CablingMapLabel = ""; 
+  CablingMapLabel = config_.getParameter<std::string> ("CablingMapLabel");
```

Comment:
Line 102 seems to serve no purpose since `CablingMapLabel` gets re-set on the following line. If there is a good reason for line 102, please add a comment there explaining the assignment. Otherwise, just delete this unnecessary assignment.